README.md

```diff
@@ -1 +1,93 @@
+
+![rosgraph of autoware.auto](content/img/rosgraph_autoware.auto.png)
+
+Future reference systems could be proposed that are more complex using the same basic node building blocks developed for this first scenario.
+
+## Supported Platforms
+
+To enable as many people as possible to replicate this reference system, the platform(s) were chosen to be easily accessible (inexpensive, high volume), has lots of documentation / community use and will be supported well into the future.
+
+Platforms were not chosen for performance of the reference system - we know we could run “faster” with a more powerful CPU or GPU but then it would be harder for others to validate findings and test their own configurations.  Accessibility is the key here and will be considered if more platforms want to be added to this benchmark list.
+
+**Platforms:** 
+ - [Raspberry Pi 4 B](https://www.raspberrypi.org/products/raspberry-pi-4-model-b/) 4 GB running the [real-time linux kernel](https://github.com/ros-realtime/rt-kernel-docker-builder)
+
+
+*Note: create an [issue](https://github.com/ros-realtime/reference-system-autoware/issues/) to add more platforms to the list, keeping in mind the above criteria*
+
+## Concept Overview
+
+Rather than trying to write code to cover all potential variations of executors, APIs, and future features we cannot even imagine today, we have chosen instead to define what we call a “reference system” based on part of a real-world system, [Autoware.Auto](https://www.autoware.org/autoware-auto).
+
+The above node graph can be boiled down to only a handful of node "types" that are repicated to make this complex system:
+
+**Node Types:**
+
+1. [**Sensor Node**](reference_system_autoware/include/reference_system_autoware/node/sensor.hpp)
+    - input node to system
+    - publishes message cyclically at some fixed frequency
+2. [**Processing Node**](reference_system_autoware/include/reference_system_autoware/node/processing.hpp)
+    - one subscriber, one publisher
+    - starts processing for N milliseconds after a message is received
+    - publishes message after processing is complete
+3. [**Fusion Node**](reference_system_autoware/include/reference_system_autoware/node/fusion.hpp)
+    - N subscribers, one publisher
+    - starts processing for N milliseconds after a message is received **from all** subscriptions
+    - publishes message after processing is complete
+4. [**Reactor Node**](reference_system_autoware/include/reference_system_autoware/node/reactor.hpp)
+    - N subscribers, one publisher
+    - starts processing for N milliseconds after a message is received **from any** single subscription
```

Comment:
@elfenpiff are you sure, because the [`reactor.hpp` node exists](https://github.com/ApexAI/reference-system-autoware/blob/feature/docs_and_tests/reference_system_autoware/include/reference_system_autoware/node/reactor.hpp) and it looks like it is [used in the `reference_system.hpp`](https://github.com/ApexAI/reference-system-autoware/blob/dbd8975c49b76f16ad5f0ba7cc5c60023111831e/reference_system_autoware/include/reference_system_autoware/reference_system.hpp#L151). what is missing?

Reply:
maybe you coded this in your sleep 😉 😄

Reply:
@flynneva I forgot to implement the number crunching for the reactor node - but it is done now.